### DISCUSSION TITLE
Thank you!!

### DISCUSSION THREAD

**Discussion by davidleebooth** [JohnXLivingston/peertube-plugin-livechat#164] [General]
This is amazing and I just wanted to say thank you for this. I use this docker image inside caprover and it works beautifully. Is there any timeline on when an updated version is released with PeerTube being 5.1 on bullseye?

**Comment by JohnXLivingston** [comment]
Hi @davidleebooth .
Are you talking about the livechat-specific docker image?
Since the v6.0.0 of the livechat plugin, you don't need this image anymore. You can use the official Peertube image. That's why I don't provide this image anymore.

There is just a bug in the building tools that prevent the livechat to work on ARM CPUs:
https://johnxlivingston.github.io/peertube-plugin-livechat/documentation/installation/cpu_compatibility/

**Reply by davidleebooth** [reply]
Hiya, awesome! so I'm guessing the best way to switch over is uninstall reinstall the plugin? Or does this involve work in docker?

**Reply by JohnXLivingston** [reply]
You just have to update the plugin, using the Peertube interface (and be sure its version is above 6.0.0). And switch to the official Peertube docker image. Nothing else to do.
